Update WebView docs to reflect added show-loading-indicator attr (#875)
Instawork/hyperview: docs/reference_web.md

A string of Javascript that gets injected into the loaded web view. Can be used to customize the loaded website.

#### `show-loading-indicator`

| Type   | Required |
| ------ | -------- |

> | string | No       |
> 
> A string specifying when to stop showing the web-view's loader. `document-only` specifies to stop loading after the document renders but before other subresources load. By default shows loader until the whole page loads.

Reviewer: let's match the doc to [your schema update](https://github.com/Instawork/hyperview/pull/872/files#diff-16e7896b76f53bc837859e9be87e4650c3e190e332e427c0f5b9898a96307964R1204-R1205):
```suggestion
| **all** (default), document-only | No       |

An attribute specifying when to stop showing the web-view's loader. `document-only` specifies to stop loading after the document renders but before other subresources load. By default (`all`), shows loader until the whole page loads.
```
